tests/UI/.eslintrc.js

```diff
@@ -50,4 +50,27 @@ module.exports = {
     'import/extensions': ['off', 'never'],
     'no-use-before-define': 0,
   },
+  overrides: [
+    {
+      files: ['*.ts'],
+      parser: '@typescript-eslint/parser',
+      plugins: ['@typescript-eslint'],
+      extends: ['prestashop', 'plugin:@typescript-eslint/eslint-recommended', 'plugin:@typescript-eslint/recommended'],
```

Comment:
@NeOMakinG Can I remove `prestashop` as it is not anymore in the `develop` branch ?